transformer/external/executabletransformer.go

```diff
@@ -204,3 +237,24 @@ func (t *Executable) uploadInput(data interface{}, inputFile string) (string, er
 	}
 	return containerInputDir, nil
 }
+
+func (t *Executable) configIO(
+	cmd environmenttypes.Command,
+	kvMap map[string]string) (
+	environmenttypes.Command,
+	[]string,
+) {
+	const dollarPrefix = "$"
+	cmdToRun := cmd
+	for envKey, value := range kvMap {
+		for index, token := range cmdToRun {
+			regexExp := regexp.MustCompile(`\$` + envKey)
```

Comment:
```suggestion
			regexExp := regexp.MustCompile(`\$({\()?` + envKey + "(}\))?")
```

Should we allow the brackets too?

Reply:
done. The above line had a minor syntax error for the quotes. Hence, I have pushed the change to PR.

Reply:
There seems to a bug in the above regex. Will try to isolate it.

Reply:
done.